data-mermaid/mermaid-webapp
# Melissa/refactor bpq

refactor BPQ to use 'alternate' abstraction for collect record forms, delete now unused abstraction

Steps to test:
@saanobhaai or people intimately familiar with this protocol should throughly test its functionality as there are no tests to test this protocol

@julia-rom please take over this and the fishbelt branch. Also feel free to rename the 'alternate' collect record form abstraction ( I didnt want to add more diff for you to review)


**NOTE:**  The base branch for this PR is develop, but its branched from the fishbelt refactor, so you can test them both from this branch.  This means the diff has both. I didnt want to set the base branch to reduce the diff since I am handing off merging to other people and its super easy up the base branch when merging.
